src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+
+    Driver create(Driver driver);
+
+    Driver get(Long id);
+
+    List<Driver> getALl();
+
+    Driver update(Driver driver);
+
+    boolean delete(Long id);
+}
```

Comment:
get() methods in Dao layer should return Optional (not in services). In the service layer better call method get() on Optional and return the object.